src/pages/Themes.tsx

```diff
@@ -41,7 +46,47 @@ const Themes: React.FC = () => {
 		searchQuery
 	);
 
-	console.log(themes);
+	useEffect(()=>{
+		//to update the current focused theme based on url and fetched themes
+		const focusedThemeId = searchParams.get('theme') || '';
+		if(focusedThemeId == ''){
+			return;
+		}
+		const focusedThemeObject = themes.find(theme => theme.id === focusedThemeId);
```

Comment:
There's a possible issue to look out for here, though we can address this in a future PR.

Users can supply a URL with a theme they wish to focus on, but if the specified theme is not loaded on the page by default, then it will not be focused due to failing this check here.

Reply:
I think a separate issue would be better for this, great thought.